test/resources/microshift-process.resource

```diff
@@ -0,0 +1,34 @@
+*** Settings ***
+Library    Process
+Library    SSHLibrary
+Library    String
+Library    OperatingSystem
+Resource    ../resources/systemd.resource
+Resource    ../resources/microshift-host.resource
+Library    ../resources/YAML.py
+
+
+*** Keywords ***
+MicroShift Version
+    [Documentation]    Run the version command
+    Login MicroShift host
+    ${version_text}    ${rc}=    Execute Command
+    ...    microshift version -o yaml
+    ...    sudo=True    return_rc=True
+    Should Be Equal As Integers    ${rc}    0
+    Should Not Be Empty    ${version_text}
+    Logout MicroShift Host
+    ${version}=  Yaml Parse  ${version_text}
+    RETURN  ${version}
+
+
+Parse Version Line
+    [Arguments]  ${line}
+    ${version_raw}=  Fetch From Right  ${line}  :
+    ${version}=  Strip String  ${version_raw}
+    RETURN  ${version}
+
+
+Restart MicroShift
+    [Documentation]  Restart the MicroShift service
+    Systemctl  restart  microshift
```

Comment:
This should be utilizing "Login MicroShift host" to ensure this restart is performed on the target MicroShift host.